Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/internal/fair_impl.h

```diff
@@ -65,7 +65,7 @@ class Fair_Polyhedron_3 {
   double weight = 0;
   Halfedge_around_vertex_circulator circ(halfedge(v,pmesh),pmesh), done(circ);
   do {
-    weight += weight_calculator.w_ij(*circ);
+    weight += CGAL::to_double(weight_calculator.w_ij(*circ));
```

Comment:
Is the `to_double()` here because the weights are templated with a number type?

Reply:
Yes, all weights are computed using strictly the `FT` type obtained from input data. Here, in the code, the double is used so I convert to be consistent.